### PR TITLE
Fixes reminder deletion return

### PR DESCRIPTION
**Pull request by elena-kolevska**

### CODE REVIEW

**Review by cicoyle — APPROVED** [review]
This doesn't per say fix a race condition, but rather fixes the return from muddying output. Change lgtm

**Comment by yaron2** [comment]
Linter

File: pkg/actors/actors.go [hunk 1]
```diff
@@ -1153,19 +1153,38 @@ func (a *actorsRuntime) ExecuteLocalOrRemoteActorReminder(ctx context.Context, r
...
+
+		select {
+		case <-ctx.Done():
+			return ctx.Err()
+		case errC := <-errCh:
+			if errC != nil {
+				fmt.Println("--- Error deleting reminder", errC)
+				return errC
+			} else {
+				fmt.Println("--- Deleted reminder", errC)
```
Comment (mikeee) [inline pkg/actors/actors.go:1184]: I don't think the nil-error `errC` should be printed here.
Are these prints going to be updated to a errorf/debugf?

Comment (yaron2) [reply pkg/actors/actors.go:1184]: Likely here for debug purposes only

**Comment by elena-kolevska** [comment]
Closing this, as @cicoyle is topuching on it as a part of a bigger PR

**Pull request closed without merging**